filebeat/crawler/prospector_log.go

```diff
@@ -140,6 +140,9 @@ func (p *ProspectorLog) scanGlob(glob string) {
 		// Track the stat data for this file for later comparison to check for
 		// rotation/etc
 		p.prospectorList[h.Path] = *h.Stat
+		// TODO: Registrar now updated also if no changes happened. Improve efficiency here
+		// Update registrar
+		p.Prospector.registrar.Persist <- h.GetState()
```

Comment:
If we reach ignore_older, we don't need to update the state. So this shouldn't be an issue?